src/app/pages/LibrarianPages/ReservationsPage/ReservationsList/index.tsx

```diff
@@ -82,7 +89,12 @@ export const ReservationsList = memo(() => {
                   </Button>
                 </Col>
                 <Col xs={6}>
-                  <Button className="btn-danger w-100 btn-sm">Deny</Button>
+                  <Button
+                    onClick={() => denyReservation({ id: reservation.id })}
+                    className="btn-danger w-100 btn-sm"
```

Comment:
It would be better if you disable the button as soon as the the librarian denied the request(You check the state of the reservation, if it is denied or rejected you disable it) the command is:
`<Button disabled= {reservation.reservationStatus === 'denied'}`